.github/workflows/go-test.yaml

```
@@ -73,7 +73,15 @@ jobs:
       - name: Rbd command
         run: |
           set -ex
-          kubectl rook-ceph rbd ls replicapool
+          SUCCEEDED=0
+          for ((i=0;i<30;i++)) ; do
+            kubectl rook-ceph rbd ls replicapool --timeout=10 || continue
```

```suggestion
           timeout 30 kubectl rook-ceph rbd ls replicapool --timeout=10 || continue
```

just adding this works?

---

You are correct. I should use `timeout`. I misunderstood that `kubectl rook-ceph` has `--timeout` option.